docs/guide/getting-started.md

```diff
@@ -26,11 +26,51 @@ if (typeof window.ethereum !== 'undefined') {
 
 You can review the full API for the `window.ethereum` object [here](./ethereum-provider.html).
 
+### Connecting to a test network.
+
+**Show test networks**<br />
+![image](https://user-images.githubusercontent.com/1038445/150907727-77fe4899-5238-4df6-8fa4-e6ae522e7d02.png)<br />
+
+![image](https://user-images.githubusercontent.com/1038445/150907769-c7ea63a8-6bd2-4f80-a376-a70243bb14b7.png)<br />
+
+**Connecting to Ropsten Test Network**<br />
+Use the network dropdown menu (which would normally be displaying Ethereum Mainnet like the image above) and select Ropsten Test Network<br />
+![image](https://user-images.githubusercontent.com/1038445/150908059-f86cd2eb-6d59-43e8-8fec-bacce425ebe0.png)<br />
+
+I recommend creating a test account just for the Ropsten Test Network. (Just give it a name)<br />
+
+![image](https://user-images.githubusercontent.com/1038445/150908294-22099c19-4b22-4587-939b-3f9f882b09e8.png)
+
+**Request 1 ETH Over the test network**<br />
+Navigate to https://faucet.metamask.io/
+Ensure you have the test account and test "Ropsten Test Network Selected"
+Click "reqest 1 ether from faucet"
```

Comment:
The line breaks here may not translate to the docs. Also, "ensure you have the test account" doesn't quite make sense. What do you think about saying:

``` suggestion
Navigate to [MetaMask's test faucet](https://faucet.metamask.io/). Then, click on "request 1 ether from faucet" and select the test account you created earlier:
```